api/config.py

```diff
@@ -13,25 +16,107 @@
 
 logging.getLogger('MARKDOWN').setLevel(logging.WARNING) # silence Markdown library
 logging.getLogger('requests').setLevel(logging.WARNING) # silence Requests library
+logging.getLogger('paste.httpserver').setLevel(logging.WARNING) # silence Paste library
```

Comment:
Is this a good idea? In dev it might be useful to see individual request logs.
Are there entirely superfluous?